samples/TerraFX/Graphics/HelloSierpinski.cs

```diff
@@ -169,7 +170,11 @@ static GraphicsMemoryRegion<IGraphicsResource> CreateTexture3DRegion(GraphicsCon
                     var y = n % TextureDz / TextureWidth;
                     var z = n / TextureDz;
 
-                    pTextureData[n] = 0xFF000000 | (z << 16) | (y << 8) | (x << 0);
+                    pTextureData[n]
+                        = (UInt32)(x << 0)     // r
+                        | (UInt32)(y << 8)     // g
+                        | (UInt32)(z << 16)    // b
+                        | 0xFF00_0000;         // a
```

Comment:
Align with the equals sign on the same line, use the keyword rather than the typename, and shift the alpha channel for consistency/readability.
```suggestion
                    pTextureData[n] = (uint)(x << 0)     // r
                                    | (uint)(y << 8)     // g
                                    | (uint)(z << 16)    // b
                                    | (uint)(0xFF << 24) // a
```

Reply:
ok